spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/GatewayApiController.java

```diff
@@ -51,4 +65,27 @@ public ApiKey findApiKeyEntityByApiKeyValue(@RequestParam("keyValue") String api
     public ServiceApi findServiceByApiKeyValue(@RequestParam("apiKeyValue") String apiKeyValue) {
         return serviceApiRepository.findByApiKeyValue(apiKeyValue);
     }
+
+    @GetMapping( "/api/gateways/{gatewayId}/apiKeys")
+    public ResponseEntity<List<ApiKey>> findApiKeyListByGatewayId(@PathVariable("gatewayId") String gatewayId) {
+        Proxy proxy = null;
+        List<ApiKey> apiKeyList;
+        String apiKeyListCreationFailedMessage = "Creation of API key list for requesting gateway failed. " +
+                "This concerns the gateway with the ID {}. ";
+        try {
+            proxy = proxyRepository.findById(gatewayId);
+            apiKeyList = proxy.getServiceApis().stream().flatMap(serviceApi -> serviceApi.getApiKeys()
```

Comment:
This and the next line are hard to read/understand. I would suggest to format it like this:
```
apiKeyList = proxy.getServiceApis().stream()
        .flatMap(
                serviceApi -> serviceApi.getApiKeys().stream())
        .collect([...])
```
Or at least add a newline before `.flatMap` and before `.collect` to make the main operations clear to the reader.

Reply:
Done.